distribution/src/main/resources/docker-compose/docker-compose-minimal.yml

```diff
@@ -29,6 +29,8 @@ services:
         -Dhxi.client.base-url=http://hxinsight-mock:8080
         -Dhxi.auth.providers.hyland-experience.token-uri=http://hxinsight-mock:8080/token
         "
+    ports:
+      - "8080:8080"
```

Comment:
Is `8000:8000` from extended service still exposed/mapped?

Reply:
Yes, it is